packages/good-design/src/theme/theme.tsx

```diff
@@ -0,0 +1,25 @@
+import { extendTheme } from "native-base";
```

Comment:
don't understand. we already have theme file at good design
why it's added not updated ?

Reply:
I believe this should/could be deleted in favor of the other of theme.ts

Reply:
removed